packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -11,25 +11,120 @@
 
   <Shw::Text::H2>Toggles</Shw::Text::H2>
 
-  <Shw::Text::H3>Button</Shw::Text::H3>
-
   <Shw::Flex as |SF|>
-    <SF.Item @label="Primary">
+    <SF.Item @label="Primary small">
+      <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @size="small" />
+    </SF.Item>
+    <SF.Item @label="Primary medium">
       <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" />
     </SF.Item>
-    <SF.Item @label="Secondary">
-      <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @color="secondary" />
+    <SF.Item @label="Primary full width">
+      <Shw::Outliner {{style width="300px"}}>
+        <Hds::Dropdown::Toggle::Button @isFullWidth={{true}} @text="Lorem ipsum" />
+      </Shw::Outliner>
     </SF.Item>
   </Shw::Flex>
+
+  <Shw::Flex as |SF|>
+    <SF.Item @label="Secondary small">
+      <Hds::Dropdown::Toggle::Button @color="secondary" @text="Lorem ipsum" @size="small" />
+    </SF.Item>
+    <SF.Item @label="Secondary medium">
+      <Hds::Dropdown::Toggle::Button @color="secondary" @text="Lorem ipsum" />
+    </SF.Item>
+    <SF.Item @label="Secondary full width">
+      <Shw::Outliner {{style width="300px"}}>
+        <Hds::Dropdown::Toggle::Button @color="secondary" @isFullWidth={{true}} @text="Lorem ipsum" />
+      </Shw::Outliner>
+    </SF.Item>
+  </Shw::Flex>
+
+  <Shw::Text::H4>With icon</Shw::Text::H4>
+
   <Shw::Flex as |SF|>
     <SF.Item @label="Primary small">
-      <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @size="small" />
+      <Hds::Dropdown::Toggle::Button @icon="hexagon" @text="Lorem ipsum" @size="small" />
+    </SF.Item>
+    <SF.Item @label="Primary medium">
+      <Hds::Dropdown::Toggle::Button @icon="hexagon" @text="Lorem ipsum" />
     </SF.Item>
+    <SF.Item @label="Primary full width">
+      <Shw::Outliner {{style width="300px"}}>
+        <Hds::Dropdown::Toggle::Button @icon="hexagon" @isFullWidth={{true}} @text="Lorem ipsum" />
+      </Shw::Outliner>
+    </SF.Item>
+  </Shw::Flex>
+  <Shw::Flex as |SF|>
     <SF.Item @label="Secondary small">
-      <Hds::Dropdown::Toggle::Button @text="Lorem ipsum" @size="small" @color="secondary" />
+      <Hds::Dropdown::Toggle::Button @color="secondary" @icon="hexagon" @text="Lorem ipsum" @size="small" />
+    </SF.Item>
+    <SF.Item @label="Secondary medium">
+      <Hds::Dropdown::Toggle::Button @color="secondary" @icon="hexagon" @text="Lorem ipsum" />
+    </SF.Item>
+    <SF.Item @label="Secondary full width">
+      <Shw::Outliner {{style width="300px"}}>
+        <Hds::Dropdown::Toggle::Button @color="secondary" @icon="hexagon" @isFullWidth={{true}} @text="Lorem ipsum" />
+      </Shw::Outliner>
     </SF.Item>
   </Shw::Flex>
 
+  <Shw::Text::H4>With count</Shw::Text::H4>
+
+  <Shw::Grid @columns="5" as |SG|>
```

Comment:
That was my initial try, but because we have so many columns for states (6) and the examples with badges take quite a bit of space, even with minimal content, I ended up moving them into their own section. We also need to test them in small variations, which is not something we test in the States section. Happy to try and squeeze them in though – don't feel strongly about it either way.